install_sct.bat

```diff
@@ -0,0 +1,66 @@
+:: Usage: install_sct.bat <version>
+:: e.g.
+::        install_sct.bat 5.5
+
+:: Turn off command echoing for nicer output
+@echo off
+
+:: Fetch version number (should be git branch, git tag, etc.)
+set git_ref=%1
+
+:: Go to user's home directory
+pushd %HOMEPATH%
+
+:: Download SCT and check out the branch requested by the user
+echo:
+echo ### Downloading SCT source code to %HOMEPATH%...
+git clone -b %git_ref% --single-branch --depth 1 https://github.com/spinalcordtoolbox/spinalcordtoolbox.git || goto error
+
+:: Create and activate virtual environment to install SCT into
+echo:
+echo ### Using Python to create virtual environment...
+cd spinalcordtoolbox
+python -m venv venv_sct || goto error
+CALL venv_sct\Scripts\activate.bat || goto error
+echo Virtual environment created and activated successfully!
+
+:: Install SCT and its requirements
+echo:
+echo ### Installing SCT and its dependencies...
+:: Pip needs to be upgraded because default p3.7 pip won't resolve dependency conflicts correctly
+python -m pip install --upgrade pip || goto error
+pip install -r requirements.txt || goto error
+pip install -e . || goto error
+
+:: Install external dependencies
+echo:
+echo ### Downloading model files and binaries...
+FOR %%D IN (PAM50 gm_model optic_models pmj_models deepseg_sc_models deepseg_gm_models deepseg_lesion_models c2c3_disc_models binaries_win) DO sct_download_data -d %%D -k || goto error
+
+:: Give further instructions that the user add the Scripts directory to their PATH
+echo:
+echo ### Installation finished!
+echo:
+echo To use SCT's command-line scripts in Command Prompt, please follow these instructions:
+echo:
+echo 1. Open the ^Start Menu -^> Type 'path' -^> Open 'Edit the System Environment Variables'
+echo 2. Click the 'Environment variables...' button
+echo 3. Under the section 'User variables for ____', highlight the 'Path' entry, then click the 'Edit...' button.
+echo 4. Click 'New', then copy and paste this directory:
+echo:
+echo    %CD%\%venv_sct\Scripts
+echo:
+echo: 5. Click 'OK' three times. You can now access SCT's scripts in the Command Prompt.
+
+:: Return to initial directory and deactivate the virtual environment
+popd
+deactivate
+
+exit /b 0

```

Comment:
I'm almost certain that [`deactivate` is a batch script](https://github.com/conda/conda/blob/master/conda/shell/condabin/deactivate.bat), and that calling a batch script without using `call` is like `goto`: [it doesn't return](https://stackoverflow.com/questions/11638705/why-does-calling-a-nested-batch-file-without-prepending-call-to-the-line-exit).

(You wouldn't run into this if you were copy-pasting each line into the command line manually, which is very confusing when testing things.)

So technically, the `exit /b 0` is currently never executed and redundant, but I suspect the intention would be clearer by saying `call deactivate` instead.

Reply:
The reason I added `exit /b 0` was because at some point in my testing, the lines from the `:error` sections were being executed? 

I'll have to double-check, but for some reason I have the memory that calling `deactivate` _did_ return. Hm.. 

(Still, I really like that you pointed this out. You're getting me to think about aspects of batch scripting + venvs on Windows that I otherwise didn't consider!)

Reply:
Ah, you're absolutely right. `deactivate` on its own doesn't return, but `call deactivate` returns. So, we should either use `deactivate` or `call deactivate ;; exit /b 0` but _not_ `deactivate ;; exit /b 0`. Subtle!